bitnami/valkey/values.yaml

```
@@ -1640,14 +1640,14 @@ metrics:
   ## @param metrics.enabled Start a sidecar prometheus exporter to expose Valkey metrics
   ##
   enabled: false
-  ## Bitnami Valkey Exporter image
-  ## ref: https://hub.docker.com/r/bitnami/valkey-exporter/tags/
-  ## @param metrics.image.registry [default: REGISTRY_NAME] Valkey Exporter image registry
-  ## @param metrics.image.repository [default: REPOSITORY_NAME/valkey-exporter] Valkey Exporter image repository
-  ## @skip metrics.image.tag Valkey Exporter image tag (immutable tags are recommended)
-  ## @param metrics.image.digest Valkey Exporter image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
-  ## @param metrics.image.pullPolicy Valkey Exporter image pull policy
-  ## @param metrics.image.pullSecrets Valkey Exporter image pull secrets
+  ## Bitnami Redis Exporter image
```

Redis is a registered mark, please update the Redis references with "Redis(R)" as in the Redis' chart

https://github.com/bitnami/charts/blob/main/bitnami/redis/values.yaml#L1776

```suggestion
  ## Bitnami Redis(R) Exporter image
```